int32 windSpeed
string date